packages/marshal/test/test-marshal-capdata.js

```diff
@@ -154,29 +154,28 @@ test('unserialize errors', t => {
 });
 
 test('unserialize extended errors', t => {
+  if (typeof AggregateError === 'undefined') {
+    t.pass('skip test on platforms prior to AggregateError');
+    return;
+  }
```

Comment:
This seems too disruptive; we should only skip the narrow subset of test logic that won't work as expected.